components/perl/Mail-SPF/perl-integrate-module.conf

```diff
@@ -0,0 +1,28 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Friedrich Kink
+#
+
+%hook-no-license%
+printf "%s\n" "$COPYRIGHT" > "$DISTRIBUTION.license"
+LICENSE="$DISTRIBUTION license"
```

Comment:
Please wait until #9451 is merged, then rebase on top of it, remove this line and re-run the `perl-integrate-module` script.  Thanks.